lib/python/ensembl/brc4/runnable/download_assembly_data.py

```diff
@@ -101,21 +107,24 @@ def md5_files(self, dl_dir: str) -> bool:
         print("File sums from %s: %d" % (md5_path, len(sums)))
         
         for dl_file, checksum in sums.items():
-            file_path = os.path.join(dl_dir, dl_file)
-            
-            if not os.path.isfile(file_path):
-                print("No file %s found" % file_path)
-                return False
-            
-            # Check the file checksum
-            with open(file_path, mode='rb') as f:
-                content = f.read()
-                file_sum = hashlib.md5(content).hexdigest()
-            if file_sum != checksum:
-                print("File %s checksum doesn't match" % file_path)
-                return False
-            else:
-                print("File checksum ok %s" % file_path)
+            for end in self.file_ends:
+                if dl_file.endswith(end):
+
+                    file_path = os.path.join(dl_dir, dl_file)
+                    
+                    if not os.path.isfile(file_path):
```

Comment:
I usually do
```
from os.path import join as pj
pj(dl_dir, dl_file)
```

Reply:
I find such short names hard to read (and in any case I'll try to use pathlib instead, which would get rid of the need of this function)

Reply:
Although a functionality that has been preserved and some people use, every time I get into a Python forum where this appears people suggest to not use aliases for two reasons:
1. It can make the code harder to read for people used to a different alias/no alias
2. As the system becomes larger, unless you are quite strict on what is imported, you are in for pain due to unknown name collisions

Reply:
Thank you!

Reply:
Now moved to using pathlib